akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -913,6 +931,51 @@ private[akka] object Running {
       _currentSequenceNumber
   }
 
+  // ===============================================
+
+  /** INTERNAL API */
+  @InternalApi private[akka] class WaitingForContinueUnstash(state: RunningState[S])
```

Comment:
this is needed so that we don't process any other command from the mailbox before unstashing all